test/envoye2e/stackdriver_plugin/fake_stackdriver/fake_stackdriver.go

```diff
@@ -142,10 +146,12 @@ func NewFakeStackdriver(port uint16, delay time.Duration) (*MetricServer, *Loggi
 	grpcServer := grpc.NewServer()
 	fsdms := &MetricServer{
 		delay:        delay,
+		timeSeries:   make([]*monitoringpb.TimeSeries, 0),
```

Comment:
nit: don't need make for `append`. append allocates array if necessary.

Reply:
Done.